bin/install.sh

```diff
@@ -248,41 +248,6 @@ install_os_release_vars() {
     fi
 }
 
-install_repo() {
```

Comment:
Since this is an interface change, I suggest you leave a routine like this:

install_repo() {
     install_error 'install_repo is deprecated, use install_repo_eval'
}

Reply:
2d530e0aa135d083036203f79898575570e15441